public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -234,64 +233,74 @@ export class CloudWatchDatasource extends DataSourceApi<CloudWatchQuery, CloudWa
         statsQuery: (param.statsGroups?.length ?? 0) > 0 ?? false,
       };
     });
+
+    const dataFrames = increasingInterval(100, 1000, 300).pipe(
+      concatMap(_ => this.makeLogActionRequest('GetQueryResults', queryParams)),
+      repeat(),
```

Comment:
I am wondering, aren't we duplicating the retry timeout logic between frontend and beckend part? I remember having some sort of retry/timeout in Go.